src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedPrimaryConstructor.cs

```diff
@@ -28,9 +28,13 @@ internal sealed class SynthesizedPrimaryConstructor : SourceConstructorSymbolBas
 
             this.MakeFlags(
                 MethodKind.Constructor,
+                RefKind.None,
                 containingType.IsAbstract ? DeclarationModifiers.Protected : DeclarationModifiers.Public,
                 returnsVoid: true,
+                hasAnyBody: false,
+                isExpressionBodied: false,
                 isExtensionMethod: false,
+                isVarArg: false,
```

Comment:
>isVarArg: false, [](http://example.com/codeflow?start=16&length=16)

This doesn't look right. Shouldn't this be based on what is in the parameters list?

Reply:
Yup.  Was done in the original work that reverted.  Didn't get around to porting that back.  Sorry.